tests/scenario/test_status.py

```diff
@@ -135,7 +133,15 @@ def test_status_check_ready(ctx, tmp_path):
         state = State(
             relations=[Relation("tempo-cluster", remote_app_data=db)],
             containers=[
-                Container("tempo", can_connect=True, mounts={"cfg": Mount(CONFIG_FILE, cfg_file)})
+                Container(
```

Comment:
Nit: this could go to conftest.

Reply:
yeah, but no because not all tests require exactly the same container mock. I'd rather keep it explicit and in-place for now, at the cost of repetition